e2e/system/build_from_git_test.go

```diff
@@ -12,10 +14,14 @@ func (s *Suite) TestBuildFromGit() {
 	s.T().Parallel()
 
 	// Setup
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
```

Comment:
Why 10 mins is used? if we use `context.Background()` the test timeout is used which is by default 10 mins or whatever the user sets

Reply:
I suggest to use `context.Background()` and when running the test use `-timeout=20m` flag to increase the test timeout. that should work.

Reply:
the `20m` is an example, use any value that you think is safe to be running on most of our platforms